lib/govuk_publishing_components/presenters/attachment.rb

```diff
@@ -47,6 +47,30 @@ def alternative_format_contact_email
         attachment_data[:alternative_format_contact_email]
       end
 
+      def reference
+        reference = []
+        reference << "ISBN #{attachment_data[:isbn]}" if attachment_data[:isbn].present?
+        reference << attachment_data[:unique_reference] if attachment_data[:unique_reference].present?
+        reference << attachment_data[:command_paper_number] if attachment_data[:command_paper_number].present?
+        if attachment_data[:hoc_paper_number].present?
+          hoc_reference = "HC #{attachment_data[:hoc_paper_number]}"
+          hoc_reference += " #{attachment_data[:parliamentary_session]}" if attachment_data[:parliamentary_session].present?
+          reference << hoc_reference
+        end
+
+        reference.join(", ")
+      end
+
+      def unnumbered_reference
+        unnumbered_reference = "Unnumbered command paper" if attachment_data[:unnumbered_command_paper].eql?(true) && !attachment_data[:command_paper_number]
```

Comment:
Does Whitehall have any translations for these as I recall most of the other copy in this component is available in multiple languages

Reply:
Whitehall does have translations for things like “Unnumbered command paper”, but, for some reason, [only in Estonian](https://github.com/alphagov/whitehall/blob/master/config/locales/et.yml#L211).

govuk_publishing_components only has locale files for English and Welsh. I’m anticipating that adding an Estonian version will open a can of worms. For example, I don't know how the component preview generation might react in the presence of locale files which have missing keys. (I suspect nothing, but would rather not take the risk).